### DISCUSSION TITLE
Error happed after bash ./configure.sh

### DISCUSSION THREAD

**Discussion by ParthTagalpallewar** [organicmaps/organicmaps#3946] [Q&A]
![Screenshot (756)](https://user-images.githubusercontent.com/55437315/203306336-c3e16731-9cfa-4060-9750-ea56b651248a.png)
![Screenshot (757)](https://user-images.githubusercontent.com/55437315/203306348-37bda49b-83ad-4748-badf-cdf242fb8d9d.png)

**Comment by biodranik** [comment]
Please re-read the instructions in docs/install.md, you have missed one important point from there. You don't have any compiler in your PATH.

**Comment by rtsisyk** [comment]
[ParthTagalpallewar](https://github.com/ParthTagalpallewar) do you still have this issue?

**Comment by rtsisyk** [comment]
https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md has been updated.
